src/layouts/SiteLaunchPad/components/SiteLaunchInfoGathering.tsx

```diff
@@ -55,16 +72,43 @@ export const SiteLaunchInfoCollectorBody = ({
         {...register("domain", { required: true })}
       />
       {errors.domain && <Text textStyle="subhead-2">Domain is required</Text>}
+
+      <Text textStyle="subhead-1" mt="1.5rem">
+        Do you want to host the both the www and non-www version of your domain?
+      </Text>
+      <Text textStyle="subhead-2">
+        Select &quot;Yes&quot; if you are not sure
+      </Text>
+      <RadioGroup
+        {...register("useWww")}
+        onChange={setSelectedRadio}
+        mt="0.25rem"
+      >
```

Comment:
double checking - i haven't actually tried this but it seems like it won't work as the `Radio` has their `value` set? feel free to resolve if you've tried before/you tried and it works

Reply:
uhm they resolve to true and false uh. creating an object of 
{
"true":true,
"fasle":false
}

seems weird. Is there a better way I could have structured this?